src/toast/toast.js

```diff
@@ -137,11 +139,23 @@ class Toast extends React.Component<ToastPropsT, ToastPrivateStateT> {
 
   animateOut = (callback: () => mixed = () => {}) => {
     this.setState({isVisible: false});
+
+    let animationDuration =
```

Comment:
please separate the environment checks from runtime checks like below

```js
let animationDuration = 200;
if (__BROWSER__) {
  if (this.bodyRef && this.bodyRef.current) {
    // set animation duration
  }
}
```

Reply:
So should I also change it [here](https://github.com/uber/baseweb/blob/dae593cc719adc2ad42839df0a601b361853dede/src/toast/toast.js#L68)?
Also may I know why does it matter? Is the compiler/minifier not smart enough to truncate ternaries but can truncate if branches?

Reply:
To me it follows convention set by [C directives](https://www.techonthenet.com/c_language/directives/if.php) and my general aversion to multi-line ternary expressions, but no blocking need to change this or other code

Reply:
Ahh gotcha but I'm quite an [expressions-ov](https://fsharpforfunandprofit.com/posts/expressions-vs-statements/)[er-statements](http://blog.agiledeveloper.com/2015/08/functional-programming-favors.html) guy we could have a nice middle-ground if javascript had an expression version of if-else haha. Anyways I'll make the changes once I know where this PR is going.